source/Halibut/Transport/SecureWebSocketListener.cs

```diff
@@ -86,13 +86,11 @@ async Task Accept()
                 {
                     try
                     {
-                        var context = await listener.GetContextAsync();
+                        var context = await listener.GetContextAsync().ConfigureAwait(false);
 
                         if (context.Request.IsWebSocketRequest)
                         {
-#pragma warning disable CS4014 // Because this call is not awaited, execution of the current method continues before the call is completed
-                            HandleClient(context);
-#pragma warning restore CS4014
+                            await HandleClient(context).ConfigureAwait(false);
```

Comment:
I dont believe we want to await on this one. The comment isn't particularly helpful, but I think we deliberatively want to run `HandleClient` in parallel